.github/pr-auto-assign-config.yml

```diff
@@ -0,0 +1,51 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+
+# A number of reviewers added to the pull request
+# Set 0 to add all the reviewers (default: 0)
+numberOfReviewers: 2
+
+# A list of keywords to be skipped the process that add reviewers if pull requests include it
+skipKeywords:
+  - wip
+  - update
+  - updating
+  - bump
+
+# The action will only run if the PR meets the specified label criterias
+filterLabels:
+  # Run
+  include:
+    - bug
+    - enhancement
+    - dependencies
+  # Not Run
+  exclude:
+    - version bump
```

Comment:
What is the default logic without any label?

Reply:
If I'm to believe, the line of code that handles this case, I believe, the review auto-assignment is to be skipped. This means, that no labeling leads to no review requested. Not sure how to go about that 🤔 If I understand the logic leading into the if statement in the auto-assign code, I believe that by not adding the `include` section, the skip doesn't happen though 🤔